nevergrad/benchmark/experiments.py

```diff
@@ -35,26 +35,25 @@
 
 
 @registry.register
-def mltuning(seed: tp.Optional[int] = None, overfitting: bool=False) -> tp.Iterator[Experiment]:
+def mltuning(seed: tp.Optional[int] = None, overfitting: bool = False) -> tp.Iterator[Experiment]:
     seedg = create_seed_generator(seed)
     # Continuous case
 
     # First, a few functions with constraints.
     optims = ["Shiwa", "DE", "DiscreteOnePlusOne", "PortfolioDiscreteOnePlusOne", "CMA", "MetaRecentering",
-              "DoubleFastGADiscreteOnePlusOne", "PSO","BO"]
+              "DoubleFastGADiscreteOnePlusOne", "PSO", "BO"]
     for dimension in [None, 1, 2, 3]:
         for regressor in ["mlp", "decision_tree", "decision_tree_depth"]:
             for dataset in (["boston", "diabetes"] if dimension is None else ["artificialcos", "artificial", "artificialsquare"]):
                 function = MLTuning(regressor=regressor, data_dimension=dimension, dataset=dataset)
-                assert dataset[:10] != "artificial" or dimension is not None
```

Comment:
@teytaud checking that arguments make sense is better done in the init, so that new xp don't need to repeat those lines everytime

Reply:
Thx.